src/utils/context.jsx

```diff
@@ -26,10 +29,12 @@ const context = function context(Component, params) {
       }
       let slide = 0;
       if (this.props.params && "slide" in this.props.params) {
-        slide = this.props.params.slide;
+        slide = Number(this.props.params.slide);
```

Comment:
yeah, I think this is what @kenwheeler was pointing out to me and yeah `oneOfType` is the solution to that